b/Blender

```diff
@@ -0,0 +1,7 @@
+import Blender
+ from Blender import Scene, Text3d
```

Comment:
Are these lines really supposed to have 1 space of indent?

Reply:
According to the hello world collection yes

Reply:
I could remove them or add tabs

Reply:
Just remove the indent